code/demo/android/app/src/main/assets/orxDemoDebug.ini

```diff
@@ -1,7 +1,7 @@
-; orx - Test config file
-; Should be used with orx v.1.13+
-
-@orx.ini@ <= Loads orx.ini settings
-
-[Display]
-Title = Orx Android test (debug)
+; orx - Test config file
+; Should be used with orx v.1.13+
+
+@orx.ini@ <= Loads orx.ini settings
```

Comment:
there is no orx.ini file anymore

Reply:
Will fix that. Though it will be loaded from `libName.ini` anyway.